Lib/difflib.py

```diff
@@ -1053,8 +1053,7 @@ def IS_LINE_JUNK(line, pat=None):
     """
 
     if pat is None:
-        stripped = line.strip()
-        return stripped == '' or stripped == '#'
+        return line.strip() in ('', '#')
```

Comment:
```suggestion
        return line.strip() in '#'
```
There's no need to search over a tuple. An empty string is in '#', and so is '#', and nothing else is. But if clarity matters, keep the regexp :wink:.

Reply:
thank you